wallet/connect/metamask-sdk/index.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 1
+description: Use the SDK in your dapp.
+tags:
+  - JavaScript SDK
+  - iOS SDK
+  - Android SDK
+  - Unity SDK
+---
+
+# MetaMask SDK
+
+MetaMask SDK provides a reliable, secure, and seamless connection between your dapp and MetaMask (extension and Mobile). It supports multiple platforms, ensuring flexibility for your integration needs.
+
+Key features include:
+
+- **Multi-platform support** – Connect from web, desktop, mobile, and gaming platforms.
+- **Seamless mobile integration** – Leverage QR codes or deeplinks to establish persistent connections with MetaMask Mobile.
```

Comment:
```suggestion
- **Seamless mobile integration** – Use QR codes or deeplinks to establish persistent connections with MetaMask Mobile.
```

We try to use "use" instead of "leverage": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/leverage